pillar/test.sls

```diff
@@ -2,6 +2,13 @@ sentry_forwarder:
     port: 5010
     sampling_rate: 10
 
+elasticsearch:
+    es_cluster_name: medal-test-cluster
+    es_memory: 1g
+    es_seed_hosts:
```

Comment:
The keys are already namespaced under the `elasticsearch` key, so I don't think we need the `es_` prefixes in addition to that.